README.md

```diff
@@ -1,5 +1,39 @@
-# talkable-ios-sdk
-Talkable Integration and API for iOS devices.
+# Talkable iOS SDK
+
+[![Version](https://img.shields.io/cocoapods/v/TalkableSDK.svg?style=flat-square)](http://cocoapods.org/pods/TalkableSDK)
+[![License](https://img.shields.io/cocoapods/l/TalkableSDK.svg?style=flat-square)](http://cocoapods.org/pods/TalkableSDK)
+[![Platform](https://img.shields.io/cocoapods/p/TalkableSDK.svg?style=flat-square)](http://cocoapods.org/pods/TalkableSDK)
+
+TalkableSDK makes it easy to integrate Talkable referral functionality into iOS apps.
+
+## Installation
```

Comment:
You already have `## Installation` below

Reply:
Oh, yeah, i'l remove it

Reply:
Not fixed